docs/howto-install-learn-ocaml-client.md

```diff
@@ -0,0 +1,24 @@
+How to install the learn-ocaml client
+=====================================
+
+This section explains how to install a learn-ocaml client for students.
+
+## Manual compilation
+
+Note: you need a working ```opam``` environment with OCaml ```4.05.0```.
+
+* Install the dependencies using:
+  - if you have opam >=2.0 available:
+``
+opam switch create . --deps-only && opam install opam-installer && eval $(opam env)
+``
```

Comment:
Note that this will recompile a 4.05.0 switch anyway, so OCaml 4.05.0 is not really needed before this point.
Also, we should probably recommend to add `--locked` to the `switch create` command, as that ensures they get the same versions of dependencies as we do (avoids e.g. the issue we had at some point with incompatible versions among the  js_of_ocaml libs)